src/ensureApiEnabled.ts

```diff
@@ -0,0 +1,92 @@
+import * as _ from "lodash";
+
+import * as api from "./api";
+import * as utils from "./utils";
+
+const POLL_INTERVAL = 10000; // 10 seconds
+const POLLS_BEFORE_RETRY = 12; // Retry enabling the API after 2 minutes
+
+export async function check(
+  projectId: string,
+  apiName: string,
+  prefix: string,
+  silent: boolean
```

Comment:
add a default here of `false` - that's basically the intended behavior in js (not passing the parameter gives us undefined, which is falsey), so it will help us when converting other files that don't actually pass this parameter.